cmd/config.go

```diff
@@ -142,15 +144,22 @@ func validateConfig(opts configOptions, flags *pflag.FlagSet) error {
 	fmt.Println("Validating config with following values")
 	printValues(values)
 
-	orgID, _ := flags.GetString("org-id")
+	var orgID string
+	orgID, _ = flags.GetString("org-id")
 	if strings.TrimSpace(orgID) != "" {
-		response, err = api.ConfigQuery(opts.cl, path, orgID, nil, values)
+		orgID, _ = flags.GetString("org-id")
```

Comment:
duplicated line, I believe you want to remove the one above

Reply:
Good spot! Addressed!